editor/src/core/workers/common/concurrency-utils.ts

```diff
@@ -0,0 +1,22 @@
+import { isFeatureEnabled } from '../../../utils/feature-switches'
+
+// TODO: this will be configurable from the RYO menu
+const PARSE_CONCURRENCY = 3
+const PARSER_CONCURRENCY_FEATURE = 'Parser Concurrency'
+
+export function isConcurrencyEnabled() {
+  return isFeatureEnabled(PARSER_CONCURRENCY_FEATURE)
+}
+
+export function getParserWorkerCount() {
+  return isConcurrencyEnabled() ? PARSE_CONCURRENCY : 1
+}
+
+export function getParserChunkCount() {
+  return isConcurrencyEnabled() ? PARSE_CONCURRENCY : 1
+}
+
+// TODO: this will be configurable from the RYO menu
+export function isConcurrencyLoggingEnabled() {
+  return isConcurrencyEnabled()
```

Comment:
we should _definitely_ not be logging these by default :) logging a couple hundred lines is a surprising toll on Chrome

Reply:
fixed. this will also be configurable from the RYO menu once #6381 will be merged